frontend/src/routes/PrivateRoutes.js

```diff
@@ -4,40 +4,82 @@
  * on the right handside canvas.
  * 
  * @author Martin Wagner
- * @author Irem
+ * @author Irem Toroslu
  */
 
 import React, { Suspense, lazy } from 'react';
 import { Redirect, Route, Switch } from 'react-router-dom';
 import SLUGS from 'resources/slugs';
 import LoadingComponent from 'components/loading';
 import DetailsComponent from 'components/details/DetailsComponent';
+import { useTheme, createUseStyles } from 'react-jss';
 const ProductSolutionsServices = lazy(() =>
     import('./dynamicPaths/ProductSolutionsServicesComponent')
 );
 // importing required components
 const DashboardComponent = lazy(() => import('../components/dashboard'));
+const useStyles = createUseStyles((theme) => ({
 
+    container: {
+      display: 'flex'
+  
+    },
+    textcontent: {
+        ...theme.typography.textcontent,
+        textAlign: 'left',
+        '@media (max-width: 768px)': {
+            display: 'none',
+            
+  
+            
+        }
+    },
+  
+    subtitle: {
+        ...theme.typography.title,
+        '@media (max-width: 1080px)': {
+            marginLeft: -50,
+  
+        },
+        '@media (max-width: 468px)': {
+            fontSize: 50,
+  
+        }
+    },
+    title: {
+        ...theme.typography.title,
+        '@media (max-width: 1080px)': {
+            marginLeft: 80,
+  
+        },
+        '@media (max-width: 468px)': {
+            fontSize: 50,
+  
+        }
+    }
+  }));
```

Comment:
A loooooot of empty space ^^

```suggestion
    container: {
      display: 'flex'
    },
    textcontent: {
        ...theme.typography.textcontent,
        textAlign: 'left',
        '@media (max-width: 768px)': {
            display: 'none',      
        }
    },
    subtitle: {
        ...theme.typography.title,
        '@media (max-width: 1080px)': {
            marginLeft: -50,
        },
        '@media (max-width: 468px)': {
            fontSize: 50,
        }
    },
    title: {
        ...theme.typography.title,
        '@media (max-width: 1080px)': {
            marginLeft: 80,
        },
        '@media (max-width: 468px)': {
            fontSize: 50,
        }
    }
  }));
```